openpype/hosts/maya/plugins/publish/extract_import_reference.py

```diff
@@ -0,0 +1,160 @@
+import os
+import sys
+
+from maya import cmds
+
+import pyblish.api
+import tempfile
+
+from openpype.lib import run_subprocess
+from openpype.pipeline import publish, legacy_io
+from openpype.settings import get_project_settings
+from openpype.hosts.maya.api import lib
+
+
+def _import_reference():
+    project_name = legacy_io.active_project()
+    project_setting = get_project_settings(project_name)
+    import_reference = (
+        project_setting["deadline"]["publish"]["MayaSubmitDeadline"]["import_reference"] # noqa
+    )
+    return import_reference
```

Comment:
I am not sure if this is good idea to do it like this. First: it won't work in new publisher, second is that it will be overwritten by the plugin settings itself when (if) they are created. For example if TD would liek to control `optional` attribute.

I suggest implementing class method `apply_settings(project_setting, system_settings)`.

Should be enough to add:

```python

@classmethod
def apply_settings(cls, project_setting, system_settings):
    cls.active = project_setting["deadline"]["publish"]["MayaSubmitDeadline"]["import_reference"] # noqa
```
but later on, when other settings are defined it must handle all of them here.